### PR TITLE
refactor(schedule): fill attribute at scheduleMapper

### PR DESCRIPTION
**Pull request by ungreat**
#### Which issue(s) this PR fixes:

Fixes #

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

File: server/odc-service/src/main/java/com/oceanbase/odc/service/schedule/factory/ScheduleResponseMapperFactory.java [hunk 1]
```diff
@@ -90,6 +91,9 @@ public class ScheduleResponseMapperFactory {
     @Autowired
     private ConnectionService dataSourceService;
 
+    @Autowired
+    private ConnectionAttributeRepository attributeRepository;
```
Comment (LioRoger) [inline server/odc-service/src/main/java/com/oceanbase/odc/service/schedule/factory/ScheduleResponseMapperFactory.java:95]: I don't see any usage for this member.

Comment (ungreat) [reply server/odc-service/src/main/java/com/oceanbase/odc/service/schedule/factory/ScheduleResponseMapperFactory.java]: removed useless field

File: server/odc-service/src/main/java/com/oceanbase/odc/service/schedule/factory/ScheduleResponseMapperFactory.java [hunk 2]
```diff
@@ -438,6 +438,7 @@ private List<Database> getDatabaseByIds(Set<Long> ids) {
         Map<Long, ConnectionConfig> id2Connection =
                 dataSourceService.internalListSkipUserCheck(connectionIds, false, false)
```
Comment (guowl3) [inline server/odc-service/src/main/java/com/oceanbase/odc/service/schedule/factory/ScheduleResponseMapperFactory.java:439]: fullFillAttributes method is executed in dataSourceService.internalListSkipUserCheck

Comment (ungreat) [reply server/odc-service/src/main/java/com/oceanbase/odc/service/schedule/factory/ScheduleResponseMapperFactory.java]: may fix the same bug.

**Review by guowl3 — APPROVED** [review]
lgtm

**Pull request merged**